snapcraft/internal/project_loader/_extensions/ros2_foxy.py

```diff
@@ -26,7 +26,7 @@
 
 
 class ExtensionImpl(Extension):
-    """Drives ROS2 build and runtime environment for snap."""
+    """Drives ROS 2 build and runtime environment for snap."""
```

Comment:
ah, I see how that comment got in (for what it's worth, this is a general issue we have)

Reply:
Yeah it happens. I won't fix this one here, but no need to continue duplicating a bad pattern.